src/internal/Subject.ts

```diff
@@ -14,17 +13,23 @@ import { arrRemove } from './util/arrRemove';
  * Subject, and you can call next to feed values as well as error and complete.
  */
 export class Subject<T> extends Observable<T> implements SubscriptionLike {
-  closed = false;
+  /** @internal */
+  _closed = false;
+
+  /**
+   * Will return true if this subject has been closed and is no longer accepting new values.
+   */
+  get closed() {
+    return this._closed;
+  }
 
   private currentObservers: Observer<T>[] | null = null;
 
-  /** @deprecated Internal implementation detail, do not use directly. Will be made internal in v8. */
+  /** @internal */
   observers: Observer<T>[] = [];
-  /** @deprecated Internal implementation detail, do not use directly. Will be made internal in v8. */
```

Comment:
_Will be ~made internal~ **removed** in v8._ 😂 Sorry about that...

Reply:
Removed/internal doesn't matter. People shouldn't be using anything marked internal. Because we could break them without warning.